### PR TITLE
Merge faces4 branch into master

### PR DESCRIPTION
**Pull request by arjantijms**
Faces4 branch is stable enough now

### CODE REVIEW

**Comment by pzygielo** [comment]
Consider
- https://github.com/eclipse-ee4j/glassfish-woodstock/pull/1424

File: pom.xml [hunk 1]
```diff
@@ -189,27 +194,40 @@
...
-                    <artifactId>maven-enforcer-plugin</artifactId>
-                    <version>3.0.0-M3</version>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>3.4.1</version>
+                    <configuration>
+                        <source>11</source>
+                        <quiet>true</quiet>
+                        <detectJavaApiLink>false</detectJavaApiLink>
+                        <additionalJOption>-Xdoclint:none</additionalJOption>
+                        <additionalparam>-Xdoclint:none</additionalparam>
```
Comment (pzygielo) [inline pom.xml:223]: https://issues.apache.org/jira/browse/MJAVADOC-475

Comment (avpinchuk) [reply pom.xml:223]: Should we simply removey `additionalparam` or replace `additionalJOption` with `doclint`?

**Comment by avpinchuk** [comment]
Please consider [Fix javadoc generation #1425](https://github.com/eclipse-ee4j/glassfish-woodstock/pull/1425)

**Review by pzygielo — APPROVED** [review]

**Pull request merged**